Sources/TuistKit/Services/TestService.swift

```diff
@@ -222,7 +228,7 @@ final class TestService {
                 target: buildableTarget.target,
                 configuration: configuration,
                 skipSigning: false
-            )
+            ) + extraTestArguments
```

Comment:
I'd add `retryCount` as a parameter of the `test` method instead of reusing the `arguments`. It will result in a more readable code.

Reply:
I think you're right! Moved